ioskrew/SwiftLayout
# 2.1.0

### focused performance of library for build and runtime

- ~support macOS from 10.15 (#153)~
- remove tutorial (#158)
- rename ViewLayout.coordinators to ViewLayout.attachments and improve implementation of UIView.sublayout
- apply SL suffix to Layoutable (#159)
- test/enchance anchors (#161)
- add options to SwiftLayoutPrinter (#162)
- revert appkit supporting
- update github action
- test/performance (#165)
- add native api view test (#167)
- add run-performance.sh
- update run-performance for sorting
- update run-performance for sorting
- using build in run-performance.sh
- update run-performance
- update run-performance for test result filtering
- Feature/layout debug print (#169)
- comment snapshot test
- update test sample
- comment test from run-performance.sh
- update run-performance.sh
- update performance
- run-performance testing only performancetests
- update performance - NSLayoutConstraint+Description.shortDescription
- update performance - AnchorsBuilder.build[Block|Array]
- update performance - typecheck 50ms
- update typecheck script
- update performance - NSLayoutConstraint+Description type-check
- update performance - function body type-check 100ms
- update performance - typecheck Anchors 50ms
- update performance - some function body type-check 50ms
- cores move to Anchors
- convenience move to Anchors
- relation move to Anchors
- models move to Anchors
- layoutguide move to Anchors
- update performance test
- Activator.layoutIfNeeded run time performance improvement
- performance test accuracy up
- remove Anchors+LayoutGuide.swift
- remove constraint sorting from Anchors
- update constraint sorting from Anchors
- update how to store sublayout and anchors in ViewLayout
- update build performance of ViewLayout.sublayout(_:)
- remove script
- Removed optional in LayoutExplorer.Component.anchors
- remove ViewInformationSet
- Revert "remove ViewInformationSet"
- Remove unnecessary type validation
- add swift ui sample
- remove swift ui sample, update tests
- add archive.sh
- add xcframework
- support xcframework
- update github action command
- add SwiftLayoutWithoutAssistantsView
- add SwiftLayoutWithoutIdentifyingView
- remove ViewInfomationSet
- update archive for xcframework
- LayoutElements - create viewDic from components
- update xcframework
- fix archive process (#170)
- remove performance test
- remove xcframework things
- Revert "feature/remove xcframework"
- Revert "Revert "feature/remove xcframework"" (#174)
- remove performance test resources
- add README_ko.md (#175)
